src/geovista/examples/vectors.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+"""Importable and runnable geovista example.
+
+Notes
+-----
+.. versionadded:: 0.4.0
+
+"""
+from __future__ import annotations
+
+import numpy as np
+import pyvista as pv
+
+import geovista
+import geovista as gv
```

Comment:
@tkoyama010 Add in the `geovista.theme` here, and this should work better, I think :+1:

Reply:
I guess it is prefer to add it. Thanks!
```suggestion
import geovista as gv
import geovista.theme
```